### ISSUE TITLE
Why I have a low accuracy when used 3dcnn in this tutorial?

### ISSUE THREAD

**Issue by ABadCandy** [regnerus/keras-alzheimers-3d-conv#1]
Hi, thanks you for sharing first!
I have about four hundreds data from ADNI and follow your tutorial with three classification(AD,MCI,CN), but I have a low accuracy finally, I worder know is there a problem with my data processing problem or model? So could your please send me a sample data for study, thanks a lot!

**Comment by regnerus** [comment]
Hi! 

First of all, this wasn't really meant as a tutorial but great that it was explanatory enough.

About the low accuracy: It could have to do with the dataset, the dataset I used which was uploaded to Kaggle but quickly taken offline again already was pre-processed. This page still contains the readme from the dataset which also lists the steps done in pre-processing. https://github.com/jbrown81/ADNI_kaggle/blob/master/adni_kaggle.ipynb

**Comment by ABadCandy** [comment]
Thanks for the reply!
      I also read the pre-processing steps and other papers before , then I use the FSL software to process the raw data and get the gray matter slices in coronal plane.
    However, when I use the pre-processed data to train a 2DCNN or 3DCNN both don't have a great result.
Maybe can you send me some sample pre-processed data to  goodcandy10@gmail.com  ? Thank you for helping!

**Comment by regnerus** [comment]
I have a zip file with the pre-processed images on Google Drive. https://drive.google.com/file/d/0B5zQ7kc5shlgRG9laXMxcnlyMk0/view?usp=sharing

Good luck!

**Comment by ABadCandy** [comment]
Thank you very much! I will try again~

**Comment by regnerus** [comment]
@ABadCandy did it work? 😄

**Comment by ABadCandy** [comment]
Yes! I used the pre-processed data you provided on 2DCNN and 3DCNN , the accuracy is ~80% and ~85% respectively. However, I also want to use the original data for comparative experiment,  Do you have the raw data downloaded from adni?（I cannot download it now）Sorry to bother you!

**Comment by regnerus** [comment]
@ABadCandy no sorry I do not have that data available but you should be able to download it somewhere right? http://adni.loni.usc.edu
